scripts/face_extractor.py

```diff
@@ -1,23 +1,26 @@
 import argparse
-from glob import glob
+import math
 import os
 import warnings
 
+from glob import glob
+
 import cv2
 import numpy as np
 from tqdm import tqdm
 
-from face_detector_threaded import FaceDetector
+from scripts.utils import check_rotation, correct_rotation
+from src.jetson.face_detector import FaceDetector
 
 """
 Given a folder of images or videos, run a face detector (literally a FaceDetector) on all images 
-or videos in the folder. Detects and crop all faces in the images or every 1/rate frames from the videos. 
+or videos in the folder. Detect and crop all faces in the images or every 1/rate frames from the videos. 
 Save the resulting crops as .jpgs in an output folder. 
 """
 
 warnings.filterwarnings('once')
 IMAGE_EXT = ['.jpg', '.JPG', '.png', '.PNG']
-VIDEO_EXT = ['.mp4', '.MP4', 'mov', '.MOV']
+VIDEO_EXT = ['.mp4', '.MP4', 'mov', '.MOV', '.avi', '.AVI']
```

Comment:
this constant needs to be moved out of each individual script and into a central location. this isnt the first time this has been defined and we're veering into the realm of introducing bugs by consistently redefining it

Reply:
Where should this go? Make some file that just contains constants?

Reply:
yeah that sounds good to me